### PR TITLE
fix : broken link

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

**Comment by eldy** [comment]
I do not experience trouble. Can you make a screenshot. Can you try to refresh you cache browser ?

**Comment by grandoc** [comment]
All images like 1downarrow.png or 1something have disappear from your theme/img that's why I had a broken link...

**Comment by eldy** [comment]
I do not experience trouble. Can you make a screenshot. Can you try to refresh you cache browser ?

File: htdocs/core/tpl/bloc_showhide.tpl.php [hunk 1]
```diff
@@ -61,7 +61,7 @@
 print '</script>'."\n";
 
 print '<div style="float:right; position: relative; top: 3px; right:5px;" id="hide-'.$blocname.'"';
-print ' class="linkobject'.($hide ? ' hideobject' : '').'">'.img_picto('', '1uparrow.png').'</div>'."\n";
```
Comment (frederic34) [inline htdocs/core/tpl/bloc_showhide.tpl.php]: @grandoc perhaps can you try `img_picto('', '1uparrow')`

**Comment by eldy** [comment]
It seems this PR was set with "Discussion" or "PR to fix" tag, but no answer nor correction was provided since a long time. So request is automatically closed.
Please reopened and add a comment if you think this is an error.

**Pull request closed without merging**